src/app/app.routes.ts

```diff
@@ -1,3 +1,12 @@
 import { Routes } from '@angular/router';
+import { LoginComponent } from './features/auth/pages/login/login.component';
+import { SignupComponent } from './features/auth/pages/signup/signup.component';
+import { FeedComponent } from './features/posts/pages/feed/feed.component';
+import { authGuard } from './core/guards/auth.guard';
 
-export const routes: Routes = [];
+export const routes: Routes = [
```

Comment:
I would add a PathMatch when the url is empty and maybe use lazy loading for feed component